packages/server/graphql/types/JiraCreateIssuePayload.ts

```diff
@@ -0,0 +1,29 @@
+import {GraphQLID, GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import StandardMutationError from './StandardMutationError'
+import {GQLContext} from '../graphql'
+import JiraIssue from './JiraIssue'
+import {resolveJiraIssue} from '../resolvers'
+
+const JiraCreateIssuePayload = new GraphQLObjectType<any, GQLContext>({
+  name: 'JiraCreateIssuePayload',
+  fields: () => ({
+    error: {
+      type: StandardMutationError
+    },
+    jiraIssue: {
+      type: JiraIssue,
+      description: 'The issue straight from Jira',
+      resolve: resolveJiraIssue
```

Comment:
is the `source` argument in `resolveJiraIssue` the same as the `data` returned in `jiraCreateIssue`?
Where is the serviceTaskId?

Reply:
The `source` arguments are different but `serviceTaskId` is only being used to grab the `cloudId` and `key` which are included in the `JiraCreateIssuePayload`. So in the resolver, I've added: 
```  
const [cloudId, issueKey] = serviceTaskId
    ? serviceTaskId.split(':')
    : [source.cloudId, source.key]
```
Perhaps this causes more confusion than it's worth but I thought the resolver could help reduce some duplicate code